pkg/storaged/pages.jsx

```diff
@@ -847,7 +847,7 @@ export const StoragePage = ({ location, plot_state }) => {
     const page = get_page_from_location(location);
 
     return (
-        <Page id="storage" className={"no-masthead-sidebar" + (client.in_anaconda_mode() ? " storage-anaconda" : "")}>
+        <Page id="storage" className={"pf-m-no-sidebar" + (client.in_anaconda_mode() ? " storage-anaconda" : "")}>
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22342-1bfa114e-20250820-113851-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)